be/src/exec/hash_table.h

```diff
@@ -354,14 +344,16 @@ class HashTable {
     const int _node_byte_size;
     // Number of non-empty buckets.  Used to determine when to grow and rehash
     int64_t _num_filled_buckets;
-    // Memory to store node data.  This is not allocated from a pool to take advantage
-    // of realloc.
-    // TODO: integrate with mem pools
-    uint8_t* _nodes;
+    // Buffer to store node data.
+    uint8_t* _current_nodes;
     // number of nodes stored (i.e. size of hash table)
     int64_t _num_nodes;
-    // max number of nodes that can be stored in '_nodes' before realloc
-    int64_t _nodes_capacity;
+    // current nodes buffer capacity
+    int64_t _current_capacity;
+    // current used
+    int64_t _current_used;
+    // total capacity
+    int64_t _total_capacity;
```

Comment:
What is the meaning of ```_total_capacity```?
It neither affect memory allocation , nor ```grow_node_array```

Reply:
I hope Node's memory allocation strategy is like STL's vector. Increase by half of the current capcity each time. Of course, it is feasible to allocate fixed-length memory each time, but it is difficult to find a suitable value.

_total_capacity is record current total capcity